StaticAnalyzer/views/android/code_analysis.py

```diff
@@ -75,9 +75,9 @@ def code_analysis(app_dir, perms, typ):
                     # print "[INFO] Doing Code Analysis on - " + jfile_path
                     relative_java_path = jfile_path.replace(java_src, '')
                     code_rule_matcher(
-                        code_findings, list(perms.keys()), dat, relative_java_path, code_rules)
+                        code_findings, perms, dat, relative_java_path, code_rules)

```

Comment:
There are certain places the keys of the permissions are needed like
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/StaticAnalyzer/views/shared_func.py#L366

changing it to dict will break it ?

Reply:
This is my problem and has been restored